e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/unissued-facility-change-from-preview-specialIssuedPermission-MIN.spec.js

```diff
@@ -22,6 +26,13 @@ let dealId;
 let token;
 let facilityOneId;
 
+const facilityEndDateEnabled = Number(Cypress.env('GEF_DEAL_VERSION')) >= 1;
+
+const MOCK_FACILITY_ONE = anUnissuedCashFacility({ facilityEndDateEnabled });
+const MOCK_FACILITY_TWO = anIssuedCashFacility({ facilityEndDateEnabled });
+const MOCK_FACILITY_THREE = anUnissuedContingentFacility({ facilityEndDateEnabled });
+const MOCK_FACILITY_FOUR = anUnissuedCashFacilityWith20MonthsOfCover({ facilityEndDateEnabled });
```

Comment:
Should we create a DRY mock/fixture for these faciliites since they're repeated a lot?

Reply:
What do you have in mind? I think this is as comonised as they can be